molecule/default/prepare.yml

```diff
@@ -0,0 +1,8 @@
+---
+- hosts: all
+  tasks:
+    - name: Hot patch redis-server service for docker operation (Debian)
+      copy:
+        src: debian-redis-server.service
```

Comment:
Is it possible to create a system override file (like with `systemctl edit`)? Or is the file so completely different that we need to replace it entirely?

Reply:
I did not know about `systemctl edit`, but also there is very little in common.

Reply:
OK, that's fine then.

FYI: `systemctl edit` creates under /etc an override.conf, that overrides specific values without replacing the entire unit file.